packages/material-ui-codemod/src/util/getJSExports.js

```diff
@@ -0,0 +1,22 @@
+import memoize from './memoize';
+import { readFileSync } from 'fs';
+import { parseSync } from '@babel/core';
```

Comment:
Can we use the jscodeshift API instead i.e. `api.jscodeshift(muiCoreIndexSource).findExportSpecifier()` (no idea if that find method exists). I guess it's not guarenteed that the parser config of the user can also read node_modules source but that's also not guarenteed in this form since we don't publish our babel config anyway.

Reply:
I tried that, got parse errors on the `export from` statements...perhaps I could build this list at compile time?

Reply:
packages/material-ui/package.json doesn't have module or main entries, otherwise i would have done what you suggested

Reply:
Since the babel config is not published, is it up to someone importing from `es/` to set the right babel options, or do you transpile at least things like `export from` statements in the build?

Reply:
@eps1lon I still want to finish up this PR, how do you want me to resolve this?